File: Ejercicios(2)/Ejemplo2/Ejemplo1/C++/Persona/Persona.cpp
#include <iostream>
#include <string>
using namespace std;

class Persona {
public:
    string nombre;
    int edad;

    void mostrar() {
        cout << "Nombre = " << nombre << "\n";
        cout << "Edad = " << edad << "\n" ;
    }
};

int main() {
    Persona p1;
    Persona p2;

    p1.nombre = "Alex";
    p1.edad = 18;

    p2.nombre = "Jhonner";
    p2.edad = 18;

    cout << "Objeto 1: \n";
    p1.mostrar();

    cout << "\nObjeto 2: \n";
    p2.mostrar();

    return 0;
}   

File: Ejercicios(2)/Ejemplo2/Ejemplo1/Python/Persona/Persona.py
class Persona: 
    def __init__(self):
        self.nombre = ""
        self.edad = 0 
        
    def mostrar(self):
        print("Nombre = ", self.nombre)
        print("Edad = ", self.edad)
        
p1 = Persona()
p2 = Persona()

p1.nombre = "Alex"
p1.edad = 18

p2.nombre = "Gabriel"
p2.edad = 19

print("Objeto 1: ")
p1.mostrar()

print("\nObjeto 2: ")
p2.mostrar()